tasks/models.py

```diff
@@ -28,6 +28,30 @@ class Task(models.Model):
     def __str__(self) -> str:
         return self.title
 
+    @classmethod
+    def create_task(cls, title, assignee, assigner, priority, status, description):
+        if not title or title == "":
+            raise ValueError("Must add title")
+        if not isinstance(priority, Priority):
+            raise ValueError("Must use Priority enum")
+        if not isinstance(status, Status):
+            raise ValueError("Must use Status enum")
+        assigner_role = assigner.role
+        assigner_team = assigner.team_id
+        assigne_team = assignee.team_id
```

Comment:
Try use `Task.objects.create()` and than check if the object was created with `isinstance`